addons/insurgents/CfgVehicles.hpp

```diff
@@ -0,0 +1,43 @@
+class CfgVehicles {
+    class I_G_Soldier_base_F;
+    class I_G_Soldier_F: I_G_Soldier_base_F {
+        class EventHandlers;
+    };
+
+    class GVAR(soldier_i): I_G_Soldier_F {
+        displayName = CSTRING(Insurgent);
+
+        scope = 2;
+        scopeCurator = 2;
+
+        uniformClass = "";
+
+        headgearProbability = 0;
+        weapons[] = {"Throw", "Put"};
+        respawnWeapons[] = {"Throw", "Put"};
+        magazines[] = {};
+        respawnMagazines[] = {};
+        linkedItems[] = {"ItemMap", "ItemCompass", "ItemWatch", "ItemRadio"};
+        respawnlinkedItems[] = {"ItemMap", "ItemCompass", "ItemWatch", "ItemRadio"};
+
+        class EventHandlers: EventHandlers {
+            init = "";
```

Comment:
Will this not break XEH?

Reply:
XEH handlers are in subclass `CBA_Extended_EventHandlers` which should be inherited.